thepeg.sh

```diff
@@ -1,7 +1,7 @@
 package: ThePEG
 version: "%(tag_basename)s"
-tag: "v2.1.2"
-source: https://github.com/alisw/thepeg
+tag: "alice/v2.2.0"
+source: https://github.com/mfasDa/thepeg
```

Comment:
Ok for testing, but for the final deployment this should be changed to a tag (not a branch) from alisw.

Reply:
Of course, preparing the branch for ThePEG 2.2.

Reply:
@mfasDa merged and tagged as v2.2.0